guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,8 +4,19 @@
 [id="katello-headline-features"]
 == Headline Features
 
-[id="katello-upgrade-warnings"]
-== Upgrade Warnings
-
-[id="katello-deprecations"]
-== Deprecations
+* Inter Server Synchronization
```

Comment:
We use headings to refer to the parts so they  show up in the table of contents
```suggestion
=== Inter Server Synchronization
```

Then I'd also think about describing the feature in the heading. Something like
```suggestion
=== Syncable format for Inter Server Synchronization
```